### ISSUE TITLE
node-gyp expects `python` to be available, default macos only has `python3` availalbe

### ISSUE THREAD

**Issue by TimoGlastra** [hyperledger/indy-vdr#151]
Best would be to also support `python3` but otherwise we need to add some documentation

**Comment by TimoGlastra** [comment]
@blu3beri any thoughts on this one?

**Comment by blu3beri** [comment]
I don't think we use node-gyp ourselves for the wrapper, ref-napi uses it. This means that if it's possible to fix, we would have to change it there, likely in the `binding.gyp`. Maybe just a discussion on GH, or something which can be linked to, will suffice for now.
